proxy_docker/app/tests/tests.sh

```diff
@@ -319,7 +353,7 @@ tests()
   # ln_newaddr
   # (GET) http://proxy:8888/ln_newaddr
 
-  echo "Testing ln_newaddr..."
+  print_title "Testing ln_newaddr..."
   response=$(curl -s proxy:8888/ln_newaddr)
   echo "response=${response}"
   address=$(echo ${response} | jq ".address")
```

Comment:
Nothing to do with your PR but it looks like CLN's newaddr used to return an `address` field but now the field name is `bech32` (default)... could you change it while you're in there?  Thanks!